cmd/launcher/options.go

```diff
@@ -178,22 +178,40 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		return nil, err
 	}
 
+	// Set control server URL and control server TLS settings based on Kolide server URL, defaulting to local server
+	controlServerURL := "localhost:3000"
+	insecureTLS := *flInsecureTLS
+	disableControlTLS := *flDisableControlTLS
+	if *flKolideServerURL == "k2device.kolide.com" {
```

Comment:
We should not allow a command line flag.

I'd probably recommend a switch statement, with the default case setting it to disabled.

Reply:
We're not allowing a command line flag for the control server -- https://github.com/kolide/launcher/blob/5f2907715481767a07f43dbbcdff45990eafcc32/cmd/launcher/options.go#L94

Reply:
`*flDisableControlTLS` I'm not sure we should expose this as a flag.

Reply:
I'll move it to the deprecated section 👍